README.md

```diff
@@ -16,82 +17,75 @@ And libraries:
+ * The bundle must have at least one channel defined in the annotations.
  * Every bundle has exactly one ClusterServiceVersion.
  * If a ClusterServiceVersion `owns` a CRD, that CRD must exist in the bundle.
- * If a ClusterServiceVersion `replaces` another, both the old and the new must exist in the package.
 
 Bundle directories are identified solely by the fact that they contain a ClusterServiceVersion, which provides an amount of freedom for layout of manifests.
 
-It's recommended to follow a layout that makes it clear which bundles are part of which package, as in [manifests](manifests):
+Check out the [operator bundle design](docs/design/operator-bundle.md) for more detail on the bundle format.
 
+# Bundle images
+
+Using [OCI spec](https://github.com/opencontainers/image-spec/blob/master/spec.md) container images as a method of storing the manifest and metadata contents of individual bundles, `opm` interacts directly with these images to generate and incrementally update the database. Once you have your [manifests defined](https://operator-framework.github.io/olm-book/docs/packaging-an-operator.html#writing-your-operator-manifests) and have created a directory in the format defined above, building the image is as simple as defining a [Dockerfile](docs/design/operator-bundle.md#Bundle-Dockerfile) and building that image:
+
+
+```sh
+podman build -t quay.io/my-container-registry-namespace/my-manifest-bundle:latest -f bundle.Dockerfile .
 ```
-manifests
-├── etcd
-│   ├── 0.6.1
-│   │   ├── etcdcluster.crd.yaml
-│   │   └── etcdoperator.clusterserviceversion.yaml
-│   ├── 0.9.0
-│   │   ├── etcdbackup.crd.yaml
-│   │   ├── etcdcluster.crd.yaml
-│   │   ├── etcdoperator.v0.9.0.clusterserviceversion.yaml
-│   │   └── etcdrestore.crd.yaml
-│   ├── 0.9.2
-│   │   ├── etcdbackup.crd.yaml
-│   │   ├── etcdcluster.crd.yaml
-│   │   ├── etcdoperator.v0.9.2.clusterserviceversion.yaml
-│   │   └── etcdrestore.crd.yaml
-│   └── etcd.package.yaml
-└── prometheus
-    ├── 0.14.0
-    │   ├── alertmanager.crd.yaml
-    │   ├── prometheus.crd.yaml
-    │   ├── prometheusoperator.0.14.0.clusterserviceversion.yaml
-    │   ├── prometheusrule.crd.yaml
-    │   └── servicemonitor.crd.yaml
-    ├── 0.15.0
-    │   ├── alertmanager.crd.yaml
-    │   ├── prometheus.crd.yaml
-    │   ├── prometheusoperator.0.15.0.clusterserviceversion.yaml
-    │   ├── prometheusrule.crd.yaml
-    │   └── servicemonitor.crd.yaml
-    ├── 0.22.2
-    │   ├── alertmanager.crd.yaml
-    │   ├── prometheus.crd.yaml
-    │   ├── prometheusoperator.0.22.2.clusterserviceversion.yaml
-    │   ├── prometheusrule.crd.yaml
-    │   └── servicemonitor.crd.yaml
-    └── prometheus.package.yaml
+
+The operator-sdk also provides tooling to generate the metadata, Dockerfile, and even build the container image itself. Some basic documentation on how this works is located in [this repository's documentation](docs/design/operator-bundle.md#Operator-SDK-CLI).
```

Comment:
Note here that "operator-sdk" reference is being removed on the bundle content validation PR. So we may need to update this later after the PR merged.

Reply:
sure, that sounds good. I think we can just remove this and then update it once the sdk pushes their own doc we can link to.